### PR TITLE
infoschema , util, autoid: Compatible with some tables in pg_catalog

### PR DESCRIPTION
**Pull request by Orion7r**
### What problem does this PR solve?
Tables in the pg_catalog system schema are used in PgBench.
Compatibility is now achieved for these tables.

### What is changed and how it works?
What's Changed:

Achieve compatibility of 'pg_catalog' system schema in PostgreSQL.
Add some system tables to the pg_catalog system schema.

How it Works:
Compatible with system tables in pg_catalog used in PgBench.

Tests 
- Unit test

### CODE REVIEW

File: infoschema/pg_catalog.go [hunk 1]
```diff
@@ -0,0 +1,37 @@
+package infoschema
```
Comment (AmoebaProtozoa) [inline infoschema/pg_catalog.go:1]: Please add license information

File: infoschema/pg_catalog_test.go [hunk 2]
```diff
@@ -0,0 +1,39 @@
+package infoschema_test
```
Comment (AmoebaProtozoa) [inline infoschema/pg_catalog_test.go:1]: Please add the correct license information

**Review by AmoebaProtozoa — CHANGES_REQUESTED** [review]

File: infoschema/pg_catalog_test.go (hunk 2 above)
Comment (Orion7r) [reply infoschema/pg_catalog_test.go:27]: already added

File: infoschema/pg_catalog.go (hunk 1 above)
Comment (Orion7r) [reply infoschema/pg_catalog.go:27]: already added

**Review by AmoebaProtozoa — APPROVED** [review]
LGTM

**Review by pupillord — APPROVED** [review]
LGTM

**Pull request merged**